internal/server/handler_v2.go

```diff
@@ -140,3 +141,21 @@ func (s *Server) V2Observation(
 	}
 	return &pbv2.ObservationResponse{}, nil
 }
+
+// V2ObservationMetric implements API for mixer.V2ObservationMetric.
+func (s *Server) V2ObservationMetric(
+	ctx context.Context, in *pbv2.ObservationRequest,
+) (*pbv2.ObservationResponse, error) {
+	if in.GetVariablesExpression() == "?" { // Get all variables for entities
```

Comment:
Given the baggage of `?` in regex, using it for get all variables feels a bit odd, is there a reason we don't want to use `*` for it?

Reply:
Per the doc do we still use `->?` for existence?

Reply:
Yea maybe "*" is more suitable...